.gitignore

```diff
@@ -17,3 +17,7 @@ src/test/test.iml
 
 *.class
 settings.xml
+
+
+src/scratch
+src/scratch/*
```

Comment:
Always include a newline at the end of the file

Was the previous pattern, `src/scratch` not sufficient to also cover this?

Reply:
Fixed in 927a2c4d7e0953cd65c38ec3e1919aef1b05c616